samples/net/sockets/http_server/src/main.c

```diff
@@ -375,8 +375,19 @@ static void setup_tls(void)
 #endif /* defined(CONFIG_NET_SAMPLE_HTTPS_SERVICE) */
 }
 
+#if defined(CONFIG_USB_DEVICE_STACK)
+int init_usb(void);
+#else
+static inline int init_usb(void)
+{
+	return 0;
+}
+#endif /* CONFIG_USB_DEVICE_STACK */
+
 int main(void)
 {
+	init_usb();
+
```

Comment:
Shorter?

```c
int main(void)
{
#if defined(CONFIG_USB_DEVICE_STACK)
	init_usb();
#endif /* CONFIG_USB_DEVICE_STACK */

```

Reply:
Why not, the code was copied verbatim from echo-server but we can do it like this too.

Reply:
Hmm, actually this is not much shorter as the function needs to be introduced too. I think I will leave this one as is for now.